cockroachcloud/export-metrics.md

```diff
@@ -8,7 +8,7 @@ cloud: true
 
 {{ site.data.products.dedicated }} users can use the [Cloud API](cloud-api.html) to configure metrics export to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [Datadog](https://www.datadoghq.com/). Once the export is configured, metrics will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud metrics sink.
 
-Exporting metrics to AWS CloudWatch is only available on {{ site.data.products.dedicated }} clusters which are hosted on AWS, and were created after August 11, 2022. Metrics export to Datadog is supported on all {{ site.data.products.dedicated }} clusters regardless of creation date.
```

Comment:
Note: the AWS limitation also still exists for metric export.

We're hoping to resolve this soon, but I think we should scope this PR to only removing the text about the GCP limitation, and then follow up with another PR once the AWS limitation is removed.